test/scientificNotation.test.js

```diff
@@ -2,11 +2,11 @@ const { scientificNotation } = require("../lib");
 
 describe("scientificNotation (all valid inputs, positive index)", () => {
   test("3 in scientific notation, no sig figs specified", () => {
-    expect(scientificNotation(3)).toBe("3 x 10<sup>5</sup>");
+    expect(scientificNotation(3)).toBe("3 x 10<sup>0</sup>");
```

Comment:
I know, this can be misleading, but this test are wrong)
Already told about it [here](https://github.com/BlakeGuilloud/ganon/pull/505/files)